report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -154,4 +154,6 @@ \section*{BenchmarkDotnet}
 
 BenchmarkDotnet\cite{Benchmarkdotnet} is a benchmarking framework allowing time measurements of operations performed in dotnet runtimes. 
 It allows for the parameterization of measured operations and provides ways to introduce warm-up operation and adjustments. 
-We use this framework to create a stress test environment that simulates stressful client behavior, and measures the response times for the system under test. 
+We use this framework to create a stress test environment that simulates stressful client behavior, and measures the response times for the system under test.
+
+We want to use results from the stress tests and benchmarks to determine whether the built-in capabilities of the HTTP server library Rocket suits the use case of the system or if the platform benefits from a multithreaded approach.
```

Comment:
```suggestion
We want to use results from the stress tests and benchmarks to determine whether the built-in capabilities of the HTTP server library Rocket suits the use case of the system or if the platform benefits from a multithreaded approach.
```
This needs to be rewritten. The bottleneck was never Rocket. The bottleneck was handling one request at a time. The only reason we switched to axum is because it was easier to share states among endpoints - not because it's faster.